src/i_oalsound.c

```diff
@@ -337,8 +336,8 @@ static void UpdateUserSoundSettings(void)
 
     if (oal_snd_module == SND_MODULE_3D)
     {
-        oal->absorption = (ALfloat)snd_absorption / 2.0f;
-        alDopplerFactor((ALfloat)snd_doppler * snd_doppler / 100.0f);
+        oal->absorption = (ALfloat)snd_absorption;
```

Comment:
Why this change (quadratic doppler to linear)?

Reply:
The velocity calculations for the doppler effect are different now (distance/time instead of "momentum"/time) so there's no need for a curve.